### PR TITLE
remove bounding_box and view_bounds variables duplication

### PR DESCRIPTION
**Pull request by Kartones**
Closes https://github.com/CartoDB/cartodb/issues/4717, https://github.com/CartoDB/cartodb/issues/5007

### CODE REVIEW

**Comment by Kartones** [comment]
@ethervoid CR please

File: app/helpers/bounding_box_helper.rb [hunk 1]
```diff
@@ -63,21 +61,21 @@ def self.current_bbox_using_stats(db,table_name)
...
-      # still (lon,lat) to be consistent with current_map_bounds_using_stats()
+      # still (lon,lat) to be consistent with current_bbox_using_stats()
       {
         max: [result[:maxx].to_f, result[:maxy].to_f],
         min: [result[:minx].to_f, result[:miny].to_f]
       }
     end
   rescue Sequel::DatabaseError => exception
-    CartoDB.notify_exception(exception, { table: table_name })
+    CartoDB.notify_exception(exception, table: table_name)
```
Comment (ethervoid) [inline app/helpers/bounding_box_helper.rb]: We have to change this to notify_error. notify_exception only let us to use user and exception :(

File: app/helpers/bounding_box_helper.rb [hunk 2]
```diff
@@ -109,28 +106,28 @@ def self.get_bbox_values(db, table_name, column_name)
 
   def self.to_polygon(minx, miny, maxx, maxy)
     return nil unless check_bounds_for(minx, miny) && check_bounds_for(maxx, maxy)
-    %Q{ST_Transform(ST_Envelope('SRID=4326;POLYGON((#{minx} #{miny}, #{minx} #{maxy}, #{maxx} #{maxy}, #{maxx} #{miny}, #{minx} #{miny}))'::geometry), 3857)}
+    %{ST_Transform(ST_Envelope('SRID=4326;POLYGON((#{minx} #{miny}, #{minx} #{maxy}, #{maxx} #{maxy}, #{maxx} #{miny}, #{minx} #{miny}))'::geometry), 3857)}
```
Comment (ethervoid) [inline app/helpers/bounding_box_helper.rb]: Que te den hound xD

File: app/helpers/bounding_box_helper.rb (hunk 1 above)
Comment (Kartones) [inline app/helpers/bounding_box_helper.rb]: right, will fix it (here I was just removing stuff to comply with Ruby rules)

**Comment by ethervoid** [comment]
Good for me :+1:

**Pull request merged**